content/c-sharp/concepts/comments/comments.md

```diff
@@ -0,0 +1,86 @@
+---
+Title: 'C# Comments'
+Description: 'A comment is a piece of text within a program that is not executed.'
+Subjects: 
+  - 'Computer Science'
+  - 'Code Foundations'  
+Tags: 
+  - 'Syntax'
+  - 'Comments'
+CatalogContent:
+  - 'learn-c-sharp'
+  - 'paths/computer-science'
+---
+
+A **comment** is a piece of text within a program that is not executed. It can be used to provide additional information to aid in understanding the code.
+
+
+### Single-line Comments
+
+#### Syntax
+
+Single-line comments are created using two consecutive forward slashes. The compiler ignores any text after `//` on the same line.
+
+```cs
+// Comment goes here
+executing code // Comment goes here
+```
+
+#### Example
```

Comment:
I have a few suggestions for this "Example" section:

- Move this entire section to _after_ the "Multi-line Comments" section.
- Change header level from 4 (#### Example) to 3 (### Example).
- Combine all of the \```cs examples into a single \```cs that demonstrates the following:
  - Single-line comments before a line of code.
  - Single-line comments after a line of code.
  - Multi-line comments.